src/oas3/__tests__/operation.test.ts

```diff
@@ -1,6 +1,72 @@
 import { transformOas3Operation } from '../operation';
 
 describe('transformOas3Operation', () => {
+  test('should return deprecated property', () => {
```

Comment:
Could we narrow down the assertion? Large snapshots are difficult to follow. :laughing: